actions/run-nextstrain-build-on-aws-batch/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: Run Nextstrain build on AWS Batch
+description: >-
+  This GitHub Actions action is intended to be called by workflows in our other
+  repos when they need to run a nextstrain build on AWS Batch.
+
+inputs:
+  runtime-envvar-names:
+    description: >-
+      A list of environment variable names for environment variables that
+      should be passed to the build. The envvar names should be separated by
+      a space, e.g. "GITHUB_RUN_ID SLACK_TOKEN SLACK_CHANNELS"
+    type: string
+    required: false
+
+  cpus:
+    description: >-
+      Number of CPUs/cores/threads/jobs to utilize at once.
+      Will be passed to the `--cpus` option for `nextstrain build` and the
+      `--cores` option for Snakemake.
+    type: number
+    default: 4
+    required: false
+
+  memory:
+    description: >-
+      Amount of memory to make available to the build. Passed to the
+      `--memory` option for `nextstrain build`.
+    type: string
+    required: false
+
+  build-args:
+    description: >-
+      Additional command-line arguments to pass to `nextstrain build` after
+      the build directory (e.g. to Snakemake).
+    type: string
+    default: ""
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - uses: nextstrain/.github/actions/setup-nextstrain-cli@master
```

Comment:
Shall we explicitly set the Python version here? Or use default?

Reply:
Not sure we need to explicitly set Python version since we are just using the Nextstrain CLI to launch the build on AWS. Maybe I'm being naive in thinking that the Python version wouldn't affect the `nextstrain build` command?

Reply:
+1 for using the default. I think `python-version` will go away from `setup-nextstrain-cli` in a future update anyhow when it switches from installing with Pip to installing with our standalone installation.